preconditions/build.gradle

```diff
@@ -3,4 +3,5 @@ apply from: "${rootDir}/gradle/publish-jar.gradle"
 dependencies {
     api project(':safe-logging')
     api 'com.google.errorprone:error_prone_annotations:2.1.3'
+    compileOnly 'com.google.code.findbugs:jsr305'
```

Comment:
This matches the NullAway repo's [recommendations](https://github.com/uber/NullAway#java-non-android).